Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -117,13 +117,19 @@ export default function widgetBehavior(publicAPI, model) {
   publicAPI.updateCursor = () => {
     switch (publicAPI.getActiveInteraction()) {
       case InteractionMethodsName.TranslateCenter:
-        model._apiSpecificRenderWindow.setCursor('move');
+        model._apiSpecificRenderWindow.setCursor(
```

Comment:
There is a default value to `cursorStyles`, but this might occur if `setCursorStyles(null)` is called explicitly to set it as null or undef. I can add an extra guard for that.